src/openMVG/features/akaze/AKAZE.hpp

```diff
@@ -55,22 +55,11 @@ struct AKAZEConfig
   {
   }
 
-  template<class Archive>
-  void serialize(Archive & ar)
```

Comment:
We have added the support to use multiple describers in the reconstruction. So we keep the describer type information during the matching, but also in Tracks and Landmarks.
But we don't keep the information about the parameters used for each method.